mibs/fujitsu/SC2.mib

```diff
@@ -0,0 +1,7322 @@
+FSC-SERVERCONTROL2-MIB DEFINITIONS ::= BEGIN
```

Comment:
This mib file should be named `FSC-SERVERCONTROL2-MIB`

Reply:
I used the same file names as fujitsu. Shall I rename it anyway?

Reply:
Yes pls

Reply:
done!